CMP1903M Object Oriented Programming A01 2223/CMP1903M A01 2223/CMP1903M A01 2223/Pack.cs

```diff
@@ -8,26 +9,89 @@ namespace CMP1903M_A01_2223
 {
     class Pack
     {
-        List<Card> pack;
+        
+        //Get, set values
+        public int typeOfShuffle { get; set; }
+        public int amount { get; set; }
+
+        //Initialise the card pack here
+        public static List<Card> _pack = new List<Card>();
+        public Card[] cardPack = _pack.ToArray();
 
         public Pack()
         {
-            //Initialise the card pack here

```

Comment:
You should initialize _pack and cardArray here and fill them with Card objects. Because right now you never create a Card objects.
Should look something like:
For every value and suit combination
    card.setValue(Value)
    card.setSuit(Suit)
    _pack.append(card)